File: apps/claimly-node-js-admin/src/api/customer/CustomerCreateInput.ts
import { ClaimCreateNestedManyWithoutCustomersInput } from "./ClaimCreateNestedManyWithoutCustomersInput";

export type CustomerCreateInput = {
  claims?: ClaimCreateNestedManyWithoutCustomersInput;
  email?: string | null;
  name?: string | null;
  phone?: string | null;
};

File: apps/claimly-server/src/APIs/Customer/Dtos/CustomerCreateInput.cs
namespace Claimly.APIs.Dtos;

public class CustomerCreateInput
{
    public List<Claim>? Claims { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Email { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public string? Phone { get; set; }

    public List<Policy>? Policies { get; set; }

    public DateTime UpdatedAt { get; set; }
}
